wled00/const.h

```diff
@@ -39,6 +39,8 @@
   #endif
 #endif
 
+#define WLED_MAX_COLOR_ORDER_MAPPINGS 10
```

Comment:
I would add different default value (5) for ESP8266 to reduce memory pressure and CPU cycles.

Reply:
Done.